staging/src/k8s.io/code-generator/cmd/validation-gen/output_tests/cross_pkg/doc.go

```diff
@@ -54,18 +54,23 @@ type T1 struct {
 	TypedefsE4 typedefs.E4 `json:"typedefse4"`
 
 	// +k8s:validateTrue="field T1.OtherString"
+	// +k8s:skipUnimported
```

Comment:
nit: would it also make sense to have a test for the failure case - eg: the case where the code panics?  Perhaps just a commented example in a test-file/doc could help users understand the purpose of the skipUnimported validator.

Reply:
I manually tested this for now.  We've got a bunch of panics in the generator that we don't yet test.  I think we should.. how do others feel about a follow up for that?

Reply:
Yeah, we don't have a good way to do failure testing yet.  Should add to the TODO if it is not there

Reply:
https://github.com/jpbetz/kubernetes/pull/110